ssz/README.md

```diff
@@ -6,25 +6,24 @@ SSZ To-Do:
 [X]Move tests to tests/ssz  
 [  ][Add full test coverage for supported types](https://github.com/ethereum/beacon_chain/issues/100)  
 [  ][Refactor ssz.py](https://github.com/ethereum/beacon_chain/issues/100)
- 
 
-## Serialization Examples 
+## Serialization Examples
 ```
 int8: 5 --> b’\x05’
 bytes: b'cow' --> b'\x00\x00\x00\x03cow'
 address: b'\x35'*20 --> b’55555555555555555555’
 hash32: b'\x35'*32 --> b’55555555555555555555555555555555’
-array: [3, 4, 5] --> [b'\x03', b'\x04', b'\x05']
+array: [3, 4, 5] --> b'\x00\x00\x00\x03\x03\x04\x05'
 ```
-## Python Usage 
+## Python Usage
 
-Serialize() takes 1 to 2 arguments. Serialize() can minimally take a value and perform standard operation, or serialize() can take a value and explicit type to shorten length of serialization. If type is not an explicitly supported type, output data will default to 4 bytes. 
+Serialize() takes 1 to 2 arguments. Serialize() can minimally take a value and perform standard operation, or serialize() can take a value and explicit type to shorten length of serialization. If type is not an explicitly supported type, output data will default to 4 bytes.
```

Comment:
nitpick:
1. Serialize() ->`serialize(val, typ)`.
2. shorten length -> shorten the length
3. perform standard -> perform the standard
4. If type is not an explicitly supported type -> If `typ` is not an explicitly supported type